### PR TITLE
Switch networking e2e to v1beta3.  Update the network-tester container.

### PR DESCRIPTION
**Pull request by brendandburns**
@lavalamp @nikhiljindal

### CODE REVIEW

**Comment by lavalamp** [comment]
LGTM

File: contrib/for-tests/network-tester/webserver.go [hunk 1]
```diff
@@ -206,9 +206,9 @@ func contactOthers(state *State) {
 	masterRO := url.URL{
 		Scheme: "http",
 		Host:   os.Getenv("KUBERNETES_RO_SERVICE_HOST") + ":" + os.Getenv("KUBERNETES_RO_SERVICE_PORT"),
-		Path:   "/api/v1beta1",
+		Path:   "/api/v1beta3",
 	}
-	client := &client.Client{client.NewRESTClient(&masterRO, "v1beta1", latest.Codec, true, 5, 10)}
+	client := &client.Client{client.NewRESTClient(&masterRO, "v1beta3", latest.Codec, false, 5, 10)}
```
Comment (nikhiljindal) [inline contrib/for-tests/network-tester/webserver.go]: How about using latest.Version here? (We are already using latest.Codec)

Comment (brendandburns) [inline contrib/for-tests/network-tester/webserver.go]: done.

**Comment by brendandburns** [comment]
comment addressed, ptal.

Thanks
--brendan

**Comment by nikhiljindal** [comment]
LGTM, thanks!
Will merge on green.

**Pull request merged**